report_thesis/src/sections/methodology.tex

```diff
@@ -97,31 +97,58 @@ \subsection{Data Preparation}
-
-An interesting discovery from our literature review was a study on the stacking and chaining of normalization methods initially aimed at classification contexts.
-Inspired by these ideas, we explored the possibility of improving model performance by optimizing the preprocessing chain for each model, per oxide, in order to determine which normalization methods would be most beneficial.
-For this purpose, we considered the following preprocessing techniques: Min-Max normalization, Z-Score normalization, Robust Scaling, Max Absolute Scaling, Norm 3, Quantile Transformer and Power Transformer.
-Additionally, we also considered dimensionality reduction techniques such as \gls{pca} and \gls{kernel-pca}.
-Through experimentation, we identified which preprocessing techniques were most effective for each model and oxide.
-Realizing the potential of this, we decided to further investigate how the predictive accuracy and robustness could be improved by combining the strengths of these models and preprocessing techniques.
-From our investigation, we determined that the stacking ensemble method seemed very promising and novel in the LIBS analysis field, leading us to select it as the primary method for our study.
-
-Stacking ensemble is analogous to the methodologies employed in the original \gls{moc} pipeline, which also tailored predictions for each oxide by blending outputs from the \gls{pls1-sm} and \gls{ica} phases, variably weighting the influence of the \gls{ica} predictions depending on the oxide.
-Unlike the \gls{moc} model, which required manual determination of the model weightings for each oxide, our method utilizes a meta learner to learn optimal parameter settings.
-Stacking ensemble is beneficial as it dynamically adapts to our dataset's characteristics without the need for domain-specific knowledge.
-This approach represents a more sophisticated method, streamlining complex model configurations and potentially enhancing predictive accuracy through dynamically learned integrations, rather than fixed presets.
-
+Choosing the right models and preprocessing techniques for \gls{libs} data analysis is a challenging task. 
+The literature suggests a variety of models and preprocessing techniques that promise to be adept at handling data that exhibits high-dimensionality, multi-collinearity, and matrix effects.
+Additionally, different machine learning models perform best on different oxides in LIBS data due to the complex nature of the data. 
+These challenges and model-specific strengths suggests that an optimal approach would involve combining multiple models. 
+This notion is supported by the advent of models such as the \gls{moc}~\cite{cleggRecalibrationMarsScience2017}, which combines the predictions of multiple models using a predefined weighting of the predictions made of each model, per-oxide.
```

Comment:
Good!

Also the SuperCam model selection paper, which quite literally uses different models per oxide. Maybe a better ref?

Reply:
> This approach improved the accuracy over the individual models, but required manual tuning of the weights for each model.

IIRC also the case for the supercam ensembles